vllm/config.py

```diff
@@ -250,6 +250,9 @@ def get_hidden_size(self) -> int:
         return self.hf_text_config.hidden_size
 
     def get_head_size(self) -> int:
+        if hasattr(self.hf_text_config, "model_type") and self.hf_text_config.model_type=='deepseek_v2':
```

Comment:
Can you add the `head_dim` to the huggingface config instead of hard coding this here?